packages/core/package.json

```diff
@@ -14,7 +14,8 @@
     "postcss-build": "postcss 'src/**/*.css' --base src -d lib && postcss 'src/**/*.css' --base src -d lib --ext 'min.css' --env 'minify'",
     "postcss-watch": "postcss 'src/**/*.css' --base src -d lib --watch",
     "storybook": "start-storybook -p 6007 --quiet",
-    "build-storybook": "build-storybook --loglevel error"
+    "build-storybook": "build-storybook --loglevel error",
+    "lint:styles": "yarn stylelint \"src/**/*.css\""
```

Comment:
Run this in the build script so errors pop up before building, like so `"build": "rimraf lib/ && yarn run lint:styles && yarn postcss-build",`

Reply:
THX 👍